### PR TITLE
Added a throwing rubbish chore

### PR DESCRIPTION
**Pull request by Fodilahy-mena**
Hi team,
Please have a look at my PR,I added one file,i would be over the moon when you change it if needs be.

Your best,
Jakaria

### CODE REVIEW

File: throwing-rabish.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Throwing rabish
```
Comment (vahona) [inline throwing-rabish.md]: Be careful , you spell wrong your file s title, do not put space in it.
any way, I will be happy if you can make more request to comment

**Review by vahona — CHANGES_REQUESTED** [review]
I think you added any file that shouldn't be here, so be careful on that.
I like your request,

**Comment by ndorfin** [comment]
Hey @Fodilahy-mena I fixed that for you!

**Comment by vahona** [comment]
Nice Jakaria

**Pull request closed without merging**